browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -114,6 +115,9 @@ WebUIController* NewWebUI(WebUI* web_ui, const GURL& url) {
   } else if (host == chrome::kChromeUISettingsHost) {
     return new BraveSettingsUI(web_ui, url.host());
   } else if (host == chrome::kChromeUINewTabHost) {
+    if (profile->IsIncognitoProfile() || profile->IsTor()) {
```

Comment:
It seems guest window still uses old private NTP.
I think it also should use new private NTP?

Reply:
added guest window flag